src/components/FlawForm.vue

```diff
@@ -241,69 +245,89 @@ const cvssString = computed(() => {
-              <button class="btn btn-secondary" @click="flaw.statement = 'Statement Text ...'">
-                Add Statement
-              </button>
-            </div>
-            <div v-if="!flaw.mitigation" class="mb-3">
-              <button class="btn btn-secondary" @click="flaw.mitigation = 'Mitigation Text ...'">
-                Add Mitigation
-              </button>
-            </div>
-          </LabelCollapsable>
+      <div class="osim-flaw-form-section border-top">
+        <LabelTextarea v-model="flaw.description" label="Comment#0" placeholder="Comment#0 ..." />
+        <LabelTextarea
+          v-if="hasSummary"
+          v-model="flaw.summary" 
+          label="Description"
+          placeholder="Description Text ..."
+        />
+        <LabelTextarea
+          v-if="hasStatement"
+          v-model="flaw.statement"
+          label="Statement"
+          placeholder="Statement Text ..."
+        />
+        <LabelTextarea
+          v-if="hasMitigation"
+          v-model="flaw.mitigation"
+          label="Mitigation"
+          placeholder="Mitigation Text ..."
+        />
+        <div class="d-flex gap-3 mb-3">
+          <button
+            v-if="!hasSummary"
+            class="btn btn-secondary"
+            @click="hasSummary = true"
+          >
+            Add Description
+          </button>
+          <button
+            v-if="!hasStatement"
+            class="btn btn-secondary"
+            @click="hasStatement = true"
+          >
+            Add Statement
+          </button>
+          <button
+            v-if="!hasMitigation"
+            class="btn btn-secondary"
+            @click="hasMitigation = true"
+          >
+            Add Mitigation
+          </button>
+        </div>
+      </div>
+      <div class="osim-flaw-form-section border-top border-bottom">
+        <div class="d-flex gap-3">
           <IssueFieldReferences
             v-model="flawReferences"
             :isEmbargoed="flaw.embargoed"
+            class="w-100"
             @reference:update="saveReferences"
             @reference:new="addBlankReference(flaw.embargoed)"
             @reference:delete="deleteReference"
           />
           <IssueFieldAcknowledgments
             v-model="flawAcknowledgments"
+            class="w-100"
             @acknowledgment:update="saveAcknowledgments"
             @acknowledgment:new="addBlankAcknowledgment(flaw.embargoed)"
             @acknowledgment:delete="deleteAcknowledgment"
           />
-
-          <LabelCollapsable label="Trackers">
-            <ul>
-              <li v-for="(tracker, trackerIndex) in trackerUuids" :key="trackerIndex">
-                <RouterLink :to="{ name: 'tracker-details', params: { id: tracker.uuid } }">
-                  {{ tracker.display }}
-                </RouterLink>
-              </li>
-            </ul>
-          </LabelCollapsable>
         </div>
+        <LabelCollapsable label="Trackers" :length="trackerUuids.length">
```

Comment:
```suggestion
        <LabelCollapsable :label="trackerUuids.length ? `Trackers: ${trackerUuids.length}` : 'Trackers'} `">
```

Reply:
This approach seems simpler to me than adding a length proper to LabelCollapsable, but adding fallthrough for disabled makes sense, or adding an isExpanded prop.